app/src/main/java/org/jellyfin/androidtv/model/itemtypes/LiftedItems.kt

```diff
@@ -3,11 +3,19 @@ package org.jellyfin.androidtv.model.itemtypes
 import org.jellyfin.apiclient.model.dto.BaseItemDto
 import org.jellyfin.apiclient.model.dto.BaseItemType
 import org.jellyfin.apiclient.model.dto.GenreDto
+import org.jellyfin.apiclient.model.entities.MediaUrl
 import org.jellyfin.apiclient.model.querying.ItemFields
 import java.util.*
 import kotlin.properties.Delegates
 
-val FIELDS_REQUIRED_FOR_LIFT = arrayOf(ItemFields.DateCreated, ItemFields.MediaSources, ItemFields.MediaStreams, ItemFields.People)
+val FIELDS_REQUIRED_FOR_LIFT = arrayOf(
+	ItemFields.DateCreated,
+	ItemFields.MediaSources,
+	ItemFields.MediaStreams,
+	ItemFields.People,
+	ItemFields.Genres,
+	ItemFields.Tags,
+	ItemFields.RemoteTrailers)
```

Comment:
This `)` annoys me lol, should be placed on the next line (_although not necessary for this PR_)